src/notification/toast-notification.component.js

```diff
@@ -249,5 +249,10 @@ angular.module( 'patternfly.notification' ).component('pfToastNotification', {
         ctrl.updateShowClose();
       }
     };
+
+    ctrl.trustAsHtml = function (html) {
+      return $sce.trustAsHtml(html);
```

Comment:
This is kind of scary. Is it really needed for most basic markup? It bypasses all sanitation of things like script tags.

Reply:
I'd feel a lot better about this if it was opt-in. For instance, using a `as-html="true"` attribute. @jeff-phillips-18 @dtaylor113 opinion?

Reply:
I would agree with that @spadgett  (and your reviews are always welcomed)

Reply:
Sounds good, opt in it is. 

The other option was to take a look at transclusion in some form, though this may have a similar outcome.